Modules/xASL_module_LongReg.m

```diff
@@ -48,7 +48,15 @@
 %% --------------------------------------------------------------
 %% Administration
 [x] = xASL_init_SubStructs(x);
-x = xASL_init_InitializeMutex(x, 'LongReg' ); % starts mutex locking process to ensure that everything will run only once
+[x, bLocked] = xASL_init_InitializeMutex(x, 'LongReg'); % starts mutex locking process to ensure that everything will run only once
```

Comment:
please remove bLocked.

Reply:
Now throughout bLocked -> x.mutex.bAnyModuleLocked

Reply:
Sounds good. Will check and test in detail.